pkg/fronius/symo.go

```diff
@@ -86,18 +88,34 @@ type (
 		Values map[string]float64
 	}
 
+	SymoMeter struct {
```

Comment:
As discussed above, I would not expose this internal type.
```suggestion
	symoMeter struct {
```